app/uk/gov/hmrc/helptosave/connectors/DESConnector.scala

```diff
@@ -31,7 +31,7 @@ import scala.concurrent.{ExecutionContext, Future}
 
 @ImplementedBy(classOf[DESConnectorImpl])
 trait DESConnector {
-  def desCorrelationId(response: HttpResponse): String = response.header("CorrelationId").getOrElse("-")
+  def desCorrelationId(response: HttpResponse): String = response.header("DesCorrelationId").getOrElse("-")
```

Comment:
does this not need to stay the same?

Reply:
Yes, made them same ie "CorrelationId" for DES and IF